powershell/tools/qltest.cmd

```diff
@@ -1,9 +1,11 @@
 @echo off
 
-if not defined CODEQL_POWERSHELL_EXTRACTOR (
-    set CODEQL_POWERSHELL_EXTRACTOR=Semmle.Extraction.PowerShell.Standalone.exe
-)
-
-type NUL && "%CODEQL_EXTRACTOR_POWERSHELL_ROOT%/tools/%CODEQL_PLATFORM%/%CODEQL_POWERSHELL_EXTRACTOR%"
+type NUL && "%CODEQL_DIST%\codeql.exe" database index-files ^
+    --prune=**/*.testproj ^
+    --include-extension=.ps1 ^
```

Comment:
Anything other extensions we should consider, like psm1 and/or psd1?

Reply:
Yeah, I think all of those are valuable to extract. I don't know if the current extractor supports them yet, though: https://github.com/microsoft/codeql/blob/22a30ab952783e6c555ce852bce7daebe12ceacb/powershell/extractor/Semmle.Extraction.PowerShell.Standalone/Options.cs#L68

We should add extraction of these other files as well, though